style.css

```diff
@@ -655,3 +694,343 @@ label {
   width: 90px;
   text-align: right;
 }
+
+.link-button:hover,
+.link-button2:hover,
+.resume:hover,
+.sub-bot:hover {
+  color: blue;
+  background: wheat;
+  border: blue;
+}
+
+@media screen and (min-width: 768px) {
+  main {
+    width: 768px;
+    align-items: center;
+  }
+
+  body {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .page1 {
+    background-image: url("/img/desktop-ver.png");
+    background-repeat: no-repeat;
+    width: 1440px;
+    height: 852px;
+    display: flex;
+    flex-direction: row;
+    justify-content: left;
+  }
+
+  .envelope {
+    margin-top: -20px;
+  }
+
+  .social-media {
+    order: 1;
+    align-self: center;
+    grid-column: 2/3;
+    grid-row: 3/4;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .main-header {
+    width: 1440px;
+    height: 72px;
+  }
```

Comment:
- [ ] Good job designing your navbar @mohisa302, Although since you used a fixed size your navbar is not responsive. As you can see your logo is hidden on a `1000px` screen size.
![Screenshot 2022-12-07 203211](https://user-images.githubusercontent.com/69077061/206499625-11be8aed-c5c7-44e6-9d9c-e62e7f91e015.png)

Reply:
- dear @Tadesse-Alemayehu, would you mind telling me in which sizes I should test my project's responsivity? I checked my web and it works entirely differently ( the images disappear and styles mashup) on 1000px, and also do I need to use platforms to test it? or I can test it with google change view on the right side of the image?

![image](https://user-images.githubusercontent.com/96848068/206515495-b5f55418-abdc-4aad-b2d7-a29f864f4041.png)

- and may I ask you please tell me how should I set the width and height of desktop mode to prevent these issues?

Reply:
Dear @mohisa302, I have mentioned to use a relative width in one of my other comments, Please use [relative width](https://www.w3schools.com/css/css_dimension.asp) to align elements in your page. something like `100%`,

Reply:
- width like `1440px` will make your webpage have a fixed width of `1440px` which overflows for any screen (laptop) with a screen size less than `1440px`.

Reply:
dear @Tadesse-Alemayehu  thanks sure. and would you mind telling me  how can I test it before request to review to make sure it works?

Reply:
Sure, Please use https://raw.githack.com/ to test the result on the online platform. Also please 🙏🙏😎😎🙏🙏 watch the video I have added to the pull request summary. It should explain most of the common matters.

Reply:
Thanks so much for your guides 🙏🙏🙏😍

Reply:
You got it, Keep up ✌️